pkg/logger/logger.go

```diff
@@ -64,6 +65,29 @@ func InitLogger() *os.File {
 	return logFile
 }
 
+// TODO : Send logs to alpamon's Logserver using a Unix domain socket
+func InitFtpLogger() {
```

Comment:
This log uses console writer, outputting only to console. Consider following existing logger setup, adjusting for dev and production environments. 
Also, using `log.Logger` as a global variable could lead to clutter.

Reply:
Will different processes started with ftp log to the same file? Not sure if this is the best approach for concurrent logging.